addons/index.js

```diff
@@ -0,0 +1,3 @@
+export {default as createAutoCorrectedDatePipe} from './createAutoCorrectedDatePipe'
```

Comment:
should the path here be `'./src/createAutoCorrectedDatePipe'.. or did you mean to put the `index.js` file in `src` dir?